tests/acceptance/features/apiOcm/share.feature

```diff
@@ -1168,11 +1168,7 @@ Feature: an user shares resources using ScienceMesh application
     When user "Alice" expires the last share of resource "textfile.txt" inside of the space "Personal"
     Then the HTTP status code should be "200"
     And using server "REMOTE"
-    When user "Brian" updates the content of federated share "textfile.txt" with "this is a new content" using the WebDAV API
-    Then the HTTP status code should be "404"
     And user "Brian" should not have a federated share "textfile.txt" shared by user "Alice" from space "Personal"
-    And using server "LOCAL"
-    And for user "Alice" the content of the file "textfile.txt" of the space "Personal" should be "ocm test"
 
```

Comment:
Same as https://github.com/owncloud/ocis/commit/2d579f760a5415059c821ba86e066ad17fd1f5a9
Somehow the fixed changes got reverted while merging the dev PR.